utils/src/test/scala/za/co/absa/enceladus/utils/implicits/StringImplicitsSuite.scala

```diff
@@ -306,4 +306,56 @@ class StringImplicitsSuite extends AnyFunSuite with Matchers {
     "X".coalesce("Y", "Z") shouldBe "X"
     "X".coalesce("") shouldBe "X"
   }
+
+
+}
+
+class StringImplicitsSuite_Extra extends AnyWordSpec with Matchers {
```

Comment:
Didn't want to rewrite the whole suite, so opted for this option. Can put it into its own file.